adafruit_lc709203f.py

```diff
@@ -55,7 +63,7 @@ class CV:
     """struct helper"""
 
     @classmethod
-    def add_values(cls, value_tuples):
+    def add_values(cls, value_tuples: Tuple[str, int, str, Union[int, None]]) -> None:
```

Comment:
* This should be wrapped in `Iterable` from `typing`
* The fourth argument should be `Optional[float]`